scripts/cron.py

```diff
@@ -0,0 +1,23 @@
+import os
+import threading
+
+import schedule
+
+from .run_signer import run_signer
+
+
+def run_threaded(job_func):
+    job_thread = threading.Thread(target=job_func)
+    job_thread.start()
+
+
+for private_key in os.getenv("PRIVATEKEY").split(","):
```

Comment:
`PRIVATE_KEYS`

Reply:
good catch

Reply:
Is there a use case for keeping the private key as an OS environment variable.  Seems like it might be an unnecessary security hole.  Any reason we cant just read it from the file and keep it in Python?

Reply:
it's loaded from the `.env` file